deploy/charts/rook-ceph-cluster/prometheus/localrules.yaml

```diff
@@ -698,37 +705,61 @@ groups:
           type: ceph_default
           oid: 1.3.6.1.4.1.50495.1.2.1.8.4
         annotations:
-          summary: Host filesystem freespace is getting low
+          summary: Host filesystem free space is low
           description: >
             Mountpoint {{ $labels.mountpoint }} on {{ $labels.nodename }}
-            will be full in less than 5 days assuming the average fill-up
-            rate of the past 48 hours.
+            will be full in less than 5 days based on the 48 hour trailing
+            fill rate.
 
       - alert: CephNodeInconsistentMTU
         expr: node_network_mtu_bytes{device!="lo"} * (node_network_up{device!="lo"} > 0) != on() group_left() (quantile(0.5, node_network_mtu_bytes{device!="lo"}))
         labels:
           severity: warning
           type: ceph_default
         annotations:
-          summary: MTU settings across Ceph hosts are inconsistent
+          summary: MTU settings across hosts are inconsistent
           description: >
             Node {{ $labels.instance }} has a different MTU size ({{ $value }})
             than the median value on device {{ $labels.device }}.
 
   - name: pools
     rules:
+      - alert: CephPoolGrowthWarning
+        expr: |
+          (predict_linear(ceph_pool_percent_used[2d], 3600 * 24 * 5) * on(pool_id)
+              group_right ceph_pool_metadata) >= 95
+        labels:
+          severity: warning
+          type: ceph_default
+          oid: 1.3.6.1.4.1.50495.1.2.1.9.2
+        annotations:
+          summary: Pool growth rate may soon exceed capacity
+          description: >
+            Pool '{{ $labels.name }}' will be full in less than 5 days
+            assuming the average fill-up rate of the past 48 hours.
       - alert: CephPoolBackfillFull
         expr: ceph_health_detail{name="POOL_BACKFILLFULL"} > 0
         labels:
           severity: warning
           type: ceph_default
         annotations:
-          summary: Freespace in a pool is too low for recovery/rebalance
+          summary: Free space in a pool is too low for recovery/rebalance
           description: >
-            A pool is approaching it's near full threshold, which will
+            A pool is approaching the near full threshold, which will
             prevent rebalance operations from completing. You should
-            consider adding more capacity to the pool.
+            consider adding more capacity.
 
+      - alert: CephPoolBackfillFull
```

Comment:
This alert looks duplicated from the one before it

Reply:
Editing flern I'm sure, pushed a fix.